src/pep8ext_naming.py

```diff
@@ -142,7 +150,7 @@ def add_options(cls, parser):
 
     @classmethod
     def parse_options(cls, options):
-        cls.ignore_names = options.ignore_names
+        cls.ignore_names = frozenset(options.ignore_names)
```

Comment:
We check this frequently so "upgrading" it to a set.

Reply:
To be clear, you're trying to avoid `O(n)` lookup and keep it to whatever the lookup complexity is for a set, right? The idea being that it's more performant?

I don't know that it's necessary, but I also don't know what people's lists look like in practice because I can't imagine folks having enough names to actually slow this down.

Reply:
That's correct. The `frozenset` approach is faster for inputs of both 5 words and 50:

```python
import random
import timeit

words = open('/usr/share/dict/words').read().splitlines()

l5 = random.sample(words, 5)
s5 = frozenset(l5)

l50 = random.sample(words, 50)
s50 = frozenset(l50)

print("5 words")
print(timeit.timeit('"word" in words', setup='from __main__ import l5 as words'))
print(timeit.timeit('"word" in words', setup='from __main__ import s5 as words'))

print("50 words")
print(timeit.timeit('"word" in words', setup='from __main__ import l50 as words'))
print(timeit.timeit('"word" in words', setup='from __main__ import s50 as words'))
```

```
5 words
0.0708019733429
0.0291919708252
50 words
0.504359006882
0.0278289318085
```

Reply:
But yes, in practice, this isn't going to make a huge difference for most people. But every little bit counts, especially when it's effectively free. 😉